lib/ruby-debug-ide/attach/util.rb

```diff
@@ -1,5 +1,36 @@
 require 'ruby-debug-ide/attach/lldb'
 require 'ruby-debug-ide/attach/gdb'
+require 'socket'
+
+def get_child_pids(pid)
+  pids = Array.new
+
+  q = Queue.new
+  q.push(pid)
+
+  while(!q.empty?) do
+    pid = q.pop
+    pids << pid
+
+    if(command_exists 'pgrep')
```

Comment:
If the command does not exist, then only one element will be added to the queue and the check will be performed once. But it does not look very nice, I agree